### ISSUE TITLE
Cannot add Nuggets or Todo list to open Incident

### ISSUE THREAD

**Issue by gregkcarson** [certsocietegenerale/FIR#112]
I have installed FIR (production) and everything seems to be working fine for the mos part however when I go to add a nugget or todo list to a case, nothing happens (no pop up window or anything). I have tried restarting services and tested in different browsers to no avail.

Here are my 'installed apps':
ubuntu@ip-10-25-2-159:/home/FIR/fir/config$ ls
base.py      __init__.pyc               production.py
base.pyc     installed_apps.txt         production.pyc
__init__.py  installed_apps.txt.sample  production.py.sample
ubuntu@ip-10-25-2-159:/home/FIR/fir/config$ cat installed_apps.txt
fir_alerting
fir_todos
fir_nuggets
fir_api

I see these tables in the database:
| fir_nuggets_nugget                                  
| fir_todos_todoitem                                  |
| fir_todos_todolisttemplate                          |
| fir_todos_todolisttemplate_concerned_business_lines |
| fir_todos_todolisttemplate_todolist    

Is it simply a matter of setting something up in the Admin panel? I looked at the FIR todo section in the admin panel but that seems just like a way of manually adding a todo item to an existing incident.

**Comment by tomchop** [comment]
If you see the add nugget button, clicking on it should make a pop-up appear. It sounds like it may be a problem with the JS. Can you tell us what browser you're using? Also, do you run into the same problem when using the development instance?

**Comment by gregkcarson** [comment]
I do see the add nugget button but nothing happens when I click it, no pop up or anything. I have tried in IE, Firefox and Chrome, all latest versions and I turned security features off to ensure nothing was blocking it. 

I have not tried the development instance.

**Comment by tomchop** [comment]
Sounds like a JS problem to me. Do you recall running the `collectstatic` command from Django, and clearing your web cache?

**Comment by gregkcarson** [comment]
Yes I ran the collectstatic command during install, I have cleared my web cache and checked again but still nothing unfortunately.

**Comment by tomchop** [comment]
Can you check your web console for any errors? (`Cmd+Alt+J` on Chrome OS X)

**Comment by gregkcarson** [comment]
Ah! 

I think you are on to something here.

![aha](https://cloud.githubusercontent.com/assets/11640674/17257415/0535aea8-5590-11e6-8297-e62da609eda4.PNG)

**Comment by gregkcarson** [comment]
Solved the issue. As expected it was just me being stupid :) In my NGINX available sites I had the /static/ location of these files set to /home/FIR/incidents/static/ instead of the root path /home/FIR/static/ which contains the necessary JS and CSS files.

I modified that file to this:
  location /static/ {
        alias /home/FIR/static/;
    }
}

Then reloaded NGINX and the 404 errors are gone and everything is working.

Thanks for helping to work through the troubleshooting with me, great tool appreciate all your hard work!
